### PR TITLE
Add theme colours to forms

### PR DESCRIPTION
**Pull request by ChinmayMhatre**
Dark/light mode for `manage-forms` https://github.com/EddieHubCommunity/LinkFree/issues/5575

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/7699"></a>

### CODE REVIEW

File: components/form/Select.js [hunk 1]
```diff
@@ -16,7 +16,7 @@ export default function Select({
       <select
         id={name}
         name={name}
-        className="mt-2 block w-full rounded-md border-0 py-1.5 pl-3 pr-10 text-gray-900 ring-1 ring-inset ring-gray-300 focus:ring-2 focus:ring-indigo-600 sm:text-sm sm:leading-6"
+        className="mt-2  text-primary-high dark:bg-transparent dark:text-white border-2 transition-all duration-250 ease-linear rounded px-6 py-2 mb-2 block w-full sm:text-sm sm:leading-6"
```
Comment (krishstwt) [inline components/form/Select.js:19]: Formatting issue
```suggestion
        className="mt-2 text-primary-high dark:bg-transparent dark:text-white border-2 transition-all duration-250 ease-linear rounded px-6 py-2 mb-2 block w-full sm:text-sm sm:leading-6"
```

**Comment by amandamartin-dev** [comment]
hey @ChinmayMhatre don't forget to link the open issue here! (if there is one)

**Comment by eddiejaoude** [comment]
I will reference my issue in the PR description as it is part of my issue and PR - thanks everyone 👍

**Review by eddiejaoude — APPROVED** [review]
Thank you 👍 

The other theme changes for the new files in `manage-forms` branch can come I another PR

**Pull request merged**